src/gentropy/config.py

```diff
@@ -282,10 +279,60 @@ class LocusToGeneConfig(StepConfig):
     wandb_run_name: str | None = None
     hf_hub_repo_id: str | None = "opentargets/locus_to_gene"
     download_from_hub: bool = True
-    write_feature_matrix: bool = True
     _target_: str = "gentropy.l2g.LocusToGeneStep"
 
 
+@dataclass
+class LocusToGeneFeatureMatrixConfig(StepConfig):
+    """Locus to gene feature matrix step configuration."""
+
+    session: Any = field(
+        default_factory=lambda: {
+            "extended_spark_conf": {
+                "spark.driver.memory": "48g",
+                "spark.executor.memory": "48g",
+                "spark.sql.shuffle.partitions": "800",
+            }
+        }
+    )
+    credible_set_path: str = MISSING
+    variant_index_path: str | None = None
+    colocalisation_path: str | None = None
+    study_index_path: str | None = None
+    gene_index_path: str | None = None
+    feature_matrix_path: str = MISSING
+    features_list: list[str] = field(
+        default_factory=lambda: [
+            # max CLPP for each (study, locus, gene) aggregating over a specific qtl type
+            "eQtlColocClppMaximum",
```

Comment:
There is no neighbourhood features for coloc. Is it because it doesn't work with old schema for current old CSs?

Reply:
That's right. I tested in a data subset that the logic was working well.
We have only trained the model on 12 features